pkg/providers/pricing/pricing.go

```diff
@@ -64,7 +65,9 @@ func NewAPI() client.PricingAPI {
 	return client.New()
 }
 
-func NewProvider(ctx context.Context, pricing client.PricingAPI, region string, startAsync <-chan struct{}) *Provider {
+func NewProvider(ctx context.Context, opts *options.Options, pricing client.PricingAPI, startAsync <-chan struct{}) *Provider {
+	region := opts.Location
```

Comment:
why not just use opts.Location in the map lookup?

Reply:
It is being used in other places (at least line 76/79) as well. But I don't think any choice matters.